openedx/core/djangolib/oauth2_retirement_utils.py

```diff
@@ -0,0 +1,41 @@
+from oauth2_provider.models import (
+    AccessToken as DOTAccessToken,
+    Application as DOTApplication,
+    Grant as DOTGrant,
+    RefreshToken as DOTRefreshToken,
+)
+from provider.oauth2.models import (
+    AccessToken as DOPAccessToken,
+    RefreshToken as DOPRefreshToken,
+    Grant as DOPGrant,
+)
+
+
+class ModelRetirer(object):
```

Comment:
@iloveagent57 Just to share context. I am the person who suggested class to group these. I don't hold a strong opinion on what @iloveagent57 is suggesting. @sandroroux It's your decision to make

Reply:
@iloveagent57 That said, I think there are value to expose two functions, one for DOT and one for DOP to give client finer control. That's how tests are structured and I liked it.

Reply:
:) Well, sorry if I suggested going back to what you originally had.  One function for each of DOT and DOP doesn't matter much to me (although I think they'll usually be called together).  I do hold a strong opinion of "don't use a class unless you need it", and I don't think you need it here, but I also don't want the perfect to be the enemy of the good.

Reply:
@iloveagent57 I like the class because it contains/protects information that I don't want public. Users cannot repurpose `delete_user_from` without repurposing `ModelRetirer`, the class groups the methods under a common concept (retiring models), and users cannot add new models to the list from another file by changing a global variable.

All that said, I'm happy to do whatever.

Reply:
👍 Fair enough, your call on this one.